dev/io.openliberty.jakarta.data.1.0_fat_tck/fat/src/io/openliberty/jakarta/data/tck/DataCoreTckLauncher.java

```diff
@@ -61,21 +55,21 @@ public static void tearDown() throws Exception {
     public void launchDataTckCore() throws Exception {
         // Test groups to run
         Map<String, String> additionalProps = new HashMap<>();
-        additionalProps.put("jakarta.tck.platform", "core");
+        additionalProps.put("jimage.dir", server.getServerSharedPath() + "jimage/output/");
+        additionalProps.put("jakarta.tck.profile", "core");
+        //FIXME Always skip signature tests since our implementation has experimental API
+        additionalProps.put("included.groups", "core & persistence & !signature");
 
         // Skip signature testing on Windows.
         // So far as I can tell the signature test plugin is not supported on this configuration
         if (System.getProperty("os.name").contains("Windows")) {
             Log.info(DataWebTckLauncher.class, "launchDataTckWeb", "Skipping Jakarta Data Signature Test on Windows");
-            additionalProps.put("test.excluded.groups", "signature");
+            additionalProps.put("included.groups", "core & persistence & !signature");
         }
 
-        //FIXME Always skip signature tests since our implementation has experimental API
-        additionalProps.put("test.excluded.groups", "signature");
-
         //TODO Remove once TCK is available from stagging repo
         additionalProps.put("jakarta.data.groupid", "io.openliberty.jakarta.data");
-        additionalProps.put("jakarta.data.tck.version", "1.0.0-112222");
+        additionalProps.put("jakarta.data.tck.version", "1.0.0-05112023");
```

Comment:
Would the timestamp be better as 20230511 so that the value can be always increasing?  This is really a comment about the code that generates it, not this pull.

Reply:
Yeah, that makes sense :) I'll do that next time we create a snapshot 👍🏼